packages/clerk-js/src/core/resources/Client.ts

```diff
@@ -110,6 +118,10 @@ export class Client extends BaseResource implements ClientResource {
     return this._basePostBypass({ body: params, path: this.path() + '/verify' });
   }
 
+  get hasAuthenticated() {
```

Comment:
Would recommend present tense instead of past-tense:
```suggestion
  get isAuthenticated() {
```

Reply:
I agree with Bryce here, present tense seems a bit better